assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -190,39 +250,36 @@ def _save(
                 self._model_dir, **self._hf_conf.get(HF_CONF.HF_TOKENIZER_ARGS.value, {})
             )
 
-        # Set experimental flag
-        if self._experimental:
-            logger.info("Experimental features enabled for MLflow conversion")
-            self._hf_conf["exp"] = True
+            # set metadata info
+            metadata = fetch_mlflow_acft_metadata(base_model_name=self._model_id,
+                                                  is_finetuned_model=False)
```

Comment:
this is needed for both OSS and HFTransformers flavor. Could you please check with @rjaincc?